docs/introduction/tutorials_intro.md

```diff
@@ -258,38 +33,31 @@ These tutorials show you how to install, export a project to and start a debuggi
 
 - [BBC micro:bit, pyOCD and GDB debugging tutorial](../debug-test/debug-microbit.html).
 - [Troubleshooting common issues](../debug-test/troubleshooting-common-issues.html).
-- [Analyzing Mbed OS crash dump tutorial](../debug-test/analyzing-mbed-os-crash-dump.html).
+- [Analyzing Mbed OS crash dump tutorial](../debug-test/analyzing-mbed-os-crash-dump.html) and [crash reporting official example](https://github.com/ARMmbed/mbed-os-example-crash-reporting)
```

Comment:
@evedon Crash Reporting is here. Where should the Sys Info one go?

Reply:
Under platform